packages/blitz-auth/src/client/index.tsx

```diff
@@ -248,6 +248,11 @@ export function getAuthValues<TProps = any>(
 function withBlitzAuthPlugin<TProps = any>(Page: ComponentType<TProps> | BlitzPage<TProps>) {
   const AuthRoot = (props: ComponentProps<any>) => {
     useSession({suspense: false})
+    const [mounted, setMounted] = useState<boolean>(false)
```

Comment:
You don't need a type here as typescript can infer that one.

```suggestion
    const [mounted, setMounted] = useState(false)
```